lexicons/app/bsky/actor/defs.json

```diff
@@ -215,6 +215,70 @@
           "description": "A list of tags which describe the account owner's interests gathered during onboarding."
         }
       }
+    },
+    "mutedWordOrTag": {
```

Comment:
is this defined as a separate ref (as opposed to just making `mutedWordsPref` an array of strings) so that it's open to adding more metadata/properties on to?

Reply:
Yep exactly e.g. timed mutes.